content/altinnstudio/_index.md

```
@@ -6,8 +6,75 @@ toc: true
 ---
 
 ### Preutfylling ved bruk av data.altinn.no og prefill.json i Altinn Studio
+Du kan forhåndsutfylle skjemaer i Altinn Studio ved å bruke data fra data.altinn.no.
+Alt du trenger er en model.prefill.json-fil og en model.cs-fil. Påse at at prefill og model-filen har matchende navn. Eksempel: `person.prefill.json` og `person.cs`.
 
-Kommer snart :)
+Du kan bruke hvilket som helst dataset fra data.altinn.no. For en oversikt over alle datasetene kan du gå hit: https://docs.data.altinn.no/datasett/. 
+
+I `person.prefill.json` fyller du ut hvilket dataset du skal bruke og hvilke felt du skal fylle. I eksempelet under bruker vi datasettet `UnitBasicInformation` og mapper feltene `BusinessAddressStreet` og `OrganizationNumber` som kommer fra endepunktet til `Email` og `OrganizationNumber` i skjemaet.
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)